### DISCUSSION TITLE
New routing example table misses parametrized route

### DISCUSSION THREAD

**Discussion by lkj4** [sveltejs/kit#6185] [Q&A] [answered]
@dummdidumm the table in https://github.com/sveltejs/kit/discussions/5774#discussioncomment-3294867 misses, e.g. something like

`/books/[bookId].svelte` to ?

**Comment by dummdidumm** [comment, marked as answer]
Updated the comment. It's `/books/[bookId]/+page.svelte`

**Reply by lkj4** [reply]
thanks!
